ibm_catalog.json

```diff
@@ -1,8 +1,8 @@
 {
   "products": [
     {
-      "name": "deploy-arch-ibm-vpc",
-      "label": "Cloud automation for VPC",
+      "name": "deploy-arch-ibm-slz-vpc",
+      "label": "Cloud foundation for VPC",
```

Comment:
@maheshwarishikha I don't think we are ready in the docs for this rename yet? If we are not ready, this should not hold up this PR and we should revert so we can merge this

Reply:
As discussed yesterday, we'll go ahead with this new name. Docs are being modified.